src/ecosystem/Ecosystem.jl

```diff
@@ -45,3 +46,119 @@ function EnvironmentalLayer(
         ),
     )
 end
+
+"""
+    truncated_standard_normal_mean(a::Float64, b::Float64)::Float64
+
+Calculates the mean of the truncated standard normal distribution. Implementation taken
+from Distribution.jl excluding unused error checks
+
+# Arguements
+- 'a' : lower bound of the truncated distribution
+- 'b' : upper bound of the truncated distribution
+
+# Returns
+- Float64, mean of the truncated standard normal distribution
+
+"""
+function truncated_standard_normal_mean(a::Float64, b::Float64)::Float64
+    if abs(a) > abs(b)
+        return - truncated_standard_normal_mean(-b, -a)
+    elseif (a == b) 
+        return a
+    end
+    mid = (a + b) / 2 
+    Δ = (b - a) * mid
+    a′ = a * invsqrt2
+    b′ = b * invsqrt2
+    if a ≤ 0 ≤ b
+        m = expm1(-Δ) * exp(-a^2 / 2) / erf(b′, a′)
+    elseif 0 < a < b
+        z = exp(-Δ) * erfcx(b′) - erfcx(a′)
+        iszero(z) && return mid
+        m = expm1(-Δ) / z
+    end
+    return clamp(m / sqrthalfπ, a, b)
+end
+
+"""
+    truncated_normal_mean(
+        normal_mean::Float64, 
+        normal_stdev::Float64, 
+        lower_bound::Float64, 
+        upper_bound::Float64
+    )::Float64   
+
+Calculates the mean of the truncated normal distribution. Implementation is taken from
+Distributions.jl excluding unused error checks
+
+# Arguements
+- 'normal_mean' : mean of the underlying (untruncated) normal distribution
+- 'normal_stdev' : standard deviation of the underlying (untruncated) normal distribution
+- 'lower_bound' : lower bound of the truncated normal distribution
+- 'upper_bound' : upper bound of the truncated normal distributionk
```

Comment:
Use backticks, not single quotations.
Just in case, it's the button directly above the tab key.

```suggestion
- `normal_mean` : mean of the underlying (untruncated) normal distribution
- `normal_stdev` : standard deviation of the underlying (untruncated) normal distribution
- `lower_bound` : lower bound of the truncated normal distribution
- `upper_bound` : upper bound of the truncated normal distributionk
```